src/components/toolbox/select/index.js

```diff
@@ -62,7 +65,7 @@ class Select extends React.Component {
         >
           {options.map((option, index) => (
             <span
-              className={`${styles.option} option`}
+              className={`${smallSizeOptions ? styles.optionSmall : styles.optionLarge} option`}
```

Comment:
```suggestion
              className={['option', styles.option, styles[size]].join(' ')}
```
💅 `smallSizeOptions` prop seems duplicate to `size` prop. IMO options should be sized based on `size`, similar to how it's handled in 
https://github.com/LiskHQ/lisk-hub/blob/8b610538bdf9bb4e6345cefff24c997eb91c400b/src/components/toolbox/buttons/css/base.css#L14-L36
and 
https://github.com/LiskHQ/lisk-hub/blob/8b610538bdf9bb4e6345cefff24c997eb91c400b/src/components/toolbox/buttons/button.js#L32-L37

Reply:
I have added a class for `.xs` and applied the size to it. Let me know if you have any other suggestion.